Sources/Mantis/CropView/CropView+Touches.swift

```diff
@@ -41,6 +41,9 @@ extension CropView {
             return
         }
         
+        // A resize event has begun by grabbing the crop UI, so notify delegate
+        delegate?.cropViewDidBeginResize(self)
```

Comment:
We need to move this line in front of 

```Swift
if touch.view is RotationDial {
     viewModel.setTouchRotationBoardStatus()
     return
}
```
because the final cropped image size may change by rotating the image.